auditbeat/tests/system/test_base.py

```diff
@@ -60,8 +60,10 @@ def test_dashboards(self):
         es = Elasticsearch([self.get_elasticsearch_url()])
         self.render_config_template(
             modules=[{
-                "name": "auditd",
-            }],
+                "name": "file_integrity",
```

Comment:
Can you fix the config for the one above too, please.

In auditbeat/Makefile I think `SYSTEM_TESTS` needs to be set to true for these to execute.